### PR TITLE
resolve 184: stop importing .css file into .scss

### PR DESCRIPTION
**Pull request by JBCSU**
# READY FOR REVIEW

# Summary
- TL;DR stop importing `normalize.css` into our `.scss`, as that functionality is deprecated

# Needed By (Date)
- Before webpack stops supporting importing `.css` files

# Urgency
- Low

# Steps to Test
1. `npm install` to remove the dependency on `normalize.css`
1. `grunt styleguide` to rebuild our .css and the styleguide
1. Verify that nothing has changed. It may be easiest to confirm that the first 143 lines of the new `decanter.css` are the same as the first 299 lines of the old one (removed all block style comments for CodeClimate).

# Affected Projects or Products
- N/A

# Associated Issues and/or People
- #184 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by JBCSU** [comment]
@sherakama I have 3 CodeClimate issues remaining for the new `_normalize.scss` file:

1. Line 58: "border-bottom: 0 is preferred over border-bottom: none" - super easy to fix
1. Line 60: "Property text-decoration already defined on line 59" - easy to fix, but I wonder why the property was defined twice in the first place?
1. Line 57: "Avoid qualifying attribute selectors with an element" - I don't think there's any way around this without changing the semantics.

It turns out all of the issues are in this block:

```css
abbr[title] {
  border-bottom: none; // 1
  text-decoration: underline; // 2
  text-decoration: underline dotted; // 2
}
```

I don't think we're ever going to emit an `<abbr>` tag, and we're even less likely to emit one with a `title` attribute. What do you think about just deleting that block from normalize? If you don't want to delete it, is there a way to retain the semantics without using an attribute qualification? If not, I can fix the other 2 errors, but I think we'll get that CodeClimate error on line 57 for ever PR for the rest of time.

**Comment by sherakama** [comment]
As this file is technically a library and we don't lint external libraries, I think we can be safe to ignore this file from codeclimate scans. Feel free to put an ignore path in to the `.codeclimate.yml` file for this.

**Comment by JBCSU** [comment]
@sherakama I've reverted `_normalize.scss` to vanilla version and CodeClimate is successfully ignoring it. Ready to merge?

**Comment by sherakama** [comment]
Wonderbar.

Thanks.
